railgun/test/integration/suite_test.go

```diff
@@ -338,6 +348,8 @@ func waitForExistingClusterReadiness(ctx context.Context, cluster ktfkind.Cluste
 			ready <- ktfkind.ProxyReadinessEvent{
 				ProxyAdminURL: proxyAdminURL,
 				ProxyURL:      proxyURL,
+				ProxyHTTPSURL: proxyHTTPSURL,
+				ProxyIP:       proxyIP,
```

Comment:
Isn't ProxyIP included within ProxyURL ?

Reply:
Yes, though not as the dedicated type, and with additional info we don't want in some cases. We want the IP alone in some cases. We need it here for the DNS overrides (to trick Go's HTTP client into sending the SNI value we want) and IMO it's more convenient for the framework to expose that than parsing it out of the string URLs.

Reply:
This only used in TestMain, agree with the convenience.